docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -26,67 +26,45 @@ You can install Grafana using our official APT repository, by downloading a `.de
 
 If you install from the APT repository, then Grafana is automatically updated every time you run `apt-get update`.
 
-| Grafana Version           | Package            | Repository                                                |
-| ------------------------- | ------------------ | --------------------------------------------------------- |
-| Grafana Enterprise        | grafana-enterprise | `https://packages.grafana.com/enterprise/deb stable main` |
-| Grafana Enterprise (Beta) | grafana-enterprise | `https://packages.grafana.com/enterprise/deb beta main`   |
-| Grafana OSS               | grafana            | `https://packages.grafana.com/oss/deb stable main`        |
-| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/deb beta main`          |
+| Grafana Version           | Package            | Repository                            |
+| ------------------------- | ------------------ | ------------------------------------- |
+| Grafana Enterprise        | grafana-enterprise | `https://apt.grafana.com stable main` |
+| Grafana Enterprise (Beta) | grafana-enterprise | `https://apt.grafana.com beta main`   |
+| Grafana OSS               | grafana            | `https://apt.grafana.com stable main` |
+| Grafana OSS (Beta)        | grafana            | `https://apt.grafana.com beta main`   |
 
 > **Note:** Grafana Enterprise is the recommended and default edition. It is available for free and includes all the features of the OSS edition. You can also upgrade to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), which has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
 
-#### To install the latest Enterprise edition:
+#### To install the latest release:
 
 ```bash
 sudo apt-get install -y apt-transport-https
 sudo apt-get install -y software-properties-common wget
-sudo wget -q -O /usr/share/keyrings/grafana.key https://packages.grafana.com/gpg.key
+sudo wget -q -O /usr/share/keyrings/grafana.key https://apt.grafana.com/gpg.key
```

Comment:
If someone installed Grafana by following the steps back when they pointed to `packages.grafana.com`, can and should they switch to using `apt.`/`rpm.grafana.com`? If so, how do they switch? Do they follow these steps and also remove the `packages...` repo?

Are we deprecating or removing `packages.grafana.com`, and if so, is that deprecation being communicated elsewhere?

Are there upgrade considerations to this change? If someone's already using `packages.grafana.com`, will upgrades still work? Will they need to add or change any repo keys?

Reply:
The original steps will keep working, and someone who deletes their old config and configures the new repository, will have everything working the same way. Since we redirect the old paths to the new one, I don't think we really need to communicate a "deprecation", right?

Reply:
> I don't think we really need to communicate a "deprecation", right?

If there are no changes in functionality or breaking changes to existing users, then it might be polite to communicate or explain the change or deprecation, but there's no requirement.

Reply:
I added a migration section to both the APT and RPM docs